esp8266/README.md

```diff
@@ -69,6 +69,32 @@ $ make PORT=/dev/ttyUSB0 deploy
 
 The image produced is `firmware-combined.bin`, to be flashed at 0x00000.
 
+First start
+-----------
+
+__Serial prompt__
+
+You can access the REPL (Python prompt) over UART resp. USB. 
```

Comment:
Yeah, and that's the problem of it - as long as you start to explain "obvious" things, you don't know where to stop, and whether you should go down to "when the Earth was yet a ball of flame ...". But then you should care about non-English readers who can't read your letters at all. So, maybe include inline English primer for few most popular languages on the planet?..

Anyway, the bottom line is that if something gets mentioned, it should be done in a way not causing confusion. "UART resp. USB" is confusing even for me. I'll take care of it.